docs/docs/tour/1-my-first-asset.md

```diff
@@ -0,0 +1,96 @@
+
+```json
+{
+  "urn": "main-postgres:my-database.orders",
+  "type": "table",
+  "service": "postgres",
+  "name": "orders",
+  "data": {
+    "database": "my-database",
+    "namespace": "main-postgres"
+  }
+}
+```
+
+- **urn** is a unique name you assign to an asset. You need to make sure you don't have a duplicate urns across all of your assets because Compass treats `urn` as an identifier of your asset. For this example, we use the following format to make sure our urn is unique, `{NAMESPACE}:{DB_NAME}.{TABLE_NAME}`. (more info about URN generation can be found [here](../guides/urn-generation))
+
+- **type** is your Asset's type. The value for type has to be recognizable by Compass. More info about Asset's Type can be found [here](../concepts/type).
+
+- **service** can be seen as the source of your asset. `service` can be anything, in this case since our `orders` table resides in `postgres`, we can just put `postgres` as the service.
+
+- **name** is the name of your asset, it does not have to be unique. We don't need to worry to get mixed up if there are other tables with the same name, `urn` will be the main identifier for your asset, that is why we need to make it unique across all of your assets.
+
+- **data** can hold your asset's extra details if there is any. In the example, we use it to store information of the **database name** and the **alias/namespace** that we use when referring the postgres instance.
+
+## 1.3 Sending your first asset to Compass
+
+Here is the asset that we built on previous section.
+
+```json
+{
+  "urn": "main-postgres:my-database.orders",
+  "type": "table",
+  "service": "postgres",
+  "name": "orders",
+  "data": {
+    "database": "my-database",
+    "namespace": "main-postgres"
+  }
+}
+```
+Let's send this into Compass so that it would be discoverable.
+
+As of now, Compass supports ingesting assets via `gRPC` and `http`. In this example, we will use `http` to send your first asset to Compass.
+Compass exposes an API `[PATCH] /v1beta1/assets` to upload your asset.
+
+```bash
+curl --location --request PATCH 'http://localhost:8080/v1beta1/assets' \
+--header 'Content-Type: application/json' \
+--header 'Compass-User-UUID: john.doe@example.com' \
+--data-raw '{
+    "asset": {
+        "urn": "main-postgres:my-database.orders",
+        "type": "table",
+        "service": "postgres",
+        "name": "orders",
+        "data": {
+            "database": "my-database",
+            "namespace": "main-postgres"
+        }
+    }
+}'
+```
+
+There are a few things to notice here:
+1. The HTTP method used is `PATCH`. This is because Compass does not have a dedicated `Create` or `Update` apis, it uses a single API to `Patch / Create` an asset instead. So when updating or patching your asset, you can use the same API.
```

Comment:
This is not entirely true now that we have a dedicated `Update` API for `PUT` calls. Might be better to say "Compass does not have a dedicated `Create` API`...